index.js

```diff
@@ -1,19 +1,20 @@
 'use strict';
 
-var resolve = require('path').resolve;
-var requireIndex = require('requireindex');
+const resolve = require('path').resolve;
+const requireIndex = require('requireindex');
 
-var rules = requireIndex(resolve(__dirname, 'lib/rules'));
-var configs = requireIndex(resolve(__dirname, 'config'));
+const rules = requireIndex(resolve(__dirname, 'lib/rules'));
+const configs = requireIndex(resolve(__dirname, 'config'));
 
-var ember = require(resolve(__dirname, 'lib/utils/ember'));
-var utils = require(resolve(__dirname, 'lib/utils/utils'));
+/* eslint-disable import/no-dynamic-require */
```

Comment:
this is literally the only place with dynamic requires, and for a clear reason. In other places we'd rather want to be explicit about this.